src/D2L.Security.OAuth2/D2L.Security.OAuth2.csproj

```diff
@@ -10,6 +10,10 @@
 		<IncludeSymbols>true</IncludeSymbols>
 	</PropertyGroup>
 
+  <ItemGroup>
+    <CompilerVisibleProperty Include="SuppressSyncGenerator" />
+  </ItemGroup>

```

Comment:
Hm, I'll read up on the docs, I was trying to make it so this was not necessary here. I'll probably need another bump of analyzers to get rid of this now.